### ISSUE TITLE
tensorboard for SQuAD training is flatlining (loss is NaN)

### ISSUE THREAD

**Issue by aurotripathy** [stanford-futuredata/dawn-bench-models#1]
Update: **CLOSING** - Possibly other unrelated error
Update: I now realize the losses are turning out to be NaNs, so its less of a tensorboard issue.
No success hooking up `tensorboard`.

I see that the code is using the `tf.summary` APIs.

Any hints?

Here's what I'm using from the `~/dawn-bench-models/tensorflow/SQuAD `folder.

`tensorboard --logdir=out/basic/`

Flags below: 
```
{ 'answer_dir': 'out/basic/00/answer',
  'answer_func': 'linear',
  'answer_path': '',
  'batch_size': 60,
  'c2q_att': True,
  'char_count_th': 50,
  'char_emb_size': 8,
  'char_out_size': 100,
  'char_vocab_size': 281,
  'cluster': True,
  'cpu_opt': False,
  'data_dir': 'data/squad',
  'data_filter': 'max',
  'debug': False,
  'decay': 0.9,
  'device': '/cpu:0',
  'device_type': 'gpu',
  'dump_answer': True,
  'dump_eval': True,
  'dump_pickle': True,
  'dynamic_att': False,
  'emb_mat': array([[-1.56657522, -0.65521857,  0.47451055, ...,  1.95406481,
        -0.19221966,  0.48359027],
       [ 0.18437961,  0.14396548,  0.50050883, ...,  0.9347548 ,
         0.29033692,  0.20650184],
       [-0.55783391, -0.22447296,  0.59943994, ...,  0.03403536,
         0.0863853 , -0.10565095],
       ...,
       [ 0.3869872 ,  0.61839947,  2.27208226, ...,  0.5445784 ,
         0.12939713, -2.41944885],
       [-1.64453907,  0.47781882,  0.87602323, ..., -0.31061648,
         0.53978496, -0.34260842],
       [ 1.02159561, -0.14910558,  1.19774686, ..., -2.11132861,
        -1.18645466, -0.52472334]]),
  'eval': True,
  'eval_dir': 'out/basic/00/eval',
  'eval_path': '',
  'eval_period': 1000,
  'filter_heights': '5',
  'finetune': False,
  'forward_name': 'single',
  'hidden_size': 100,
  'highway': True,
  'highway_num_layers': 2,
  'init_lr': 0.001,
  'input_keep_prob': 0.8,
  'keep_prob': 0.8,
  'known_if_glove': True,
  'len_opt': True,
  'load': False,
  'load_ema': True,
  'load_path': '',
  'load_step': 0,
  'log_dir': 'out/basic/00/log',
  'log_period': 100,
  'logit_func': 'tri_linear',
  'lower_word': True,
  'max_num_sents': 1,
  'max_para_size': 256,
  'max_ques_size': 30,
  'max_sent_size': 400,
  'max_to_keep': 20,
  'max_word_size': 16,
  'mode': 'train',
  'model_name': 'basic',
  'na': False,
  'num_epochs': 12,
  'num_gpus': 1,
  'num_sents_th': 8,
  'num_steps': 20000,
  'out_base_dir': 'out',
  'out_channel_dims': '100',
  'out_dir': 'out/basic/00',
  'para_size_th': 256,
  'progress': True,
  'q2c_att': True,
  'ques_size_th': 30,
  'run_id': '0',
  'save_dir': 'out/basic/00/save',
  'save_period': 1000,
  'sent_size_th': 400,
  'sh_logit_func': 'tri_linear',
  'share_cnn_weights': True,
  'share_lstm_weights': True,
  'shared_path': '',
  'single': False,
  'squash': False,
  'swap_memory': True,
  'test_num_batches': 0,
  'th': 0.5,
  'use_char_emb': True,
  'use_glove_for_unk': True,
  'use_word_emb': True,
  'val_num_batches': 100,
  'var_decay': 0.999,
  'vis': False,
  'wd': 0.0,
  'word_count_th': 10,
  'word_emb_size': 100,
  'word_size_th': 16,
  'word_vocab_size': 1229,
  'wy': False}
```
